Source/ACE/Entity/Vendor.cs

```diff
@@ -215,11 +215,15 @@ public void BuyValidateTransaction(ObjectGuid vendorid, List<ItemProfile> items,
             foreach (WorldObject wo in uqlist)
             {
                 goldcost = goldcost + (uint)Math.Ceiling(SellRate * (wo.Value ?? 0) * (wo.StackSize ?? 1) - 0.1);
+                wo.Value = wo.Value;        // Also set the stack's value for unique items, using the builtin WO calculations
```

Comment:
I find this sorta odd that these lines are required.. i would think it should have auto updated the value based on on the getter.. maybe something else is needed though..

Reply:
Yes, I tested it without these lines present; however, the value and burden of the items, regardless of stack count, didn't set themselves correctly, until the stack was manipulated.  I am wondering maybe item examine is one of the processes that acts upon the AO, instead of the WO, as you mentioned at one point.